src/test/java/com/solvd/movie/service/impl/EsMovieServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package com.solvd.movie.service.impl;
+
+import com.solvd.movie.model.EsMovie;
+import com.solvd.movie.model.criteria.SearchCriteria;
+import com.solvd.movie.persistence.EsRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+@ExtendWith(MockitoExtension.class)
+public class EsMovieServiceTest {
+
+    private final FakeMovieService fakeMovieService = new FakeMovieService();
+
+    @Mock
+    private EsRepository esRepository;
```

Comment:
the better place for fake objects will be to fake the repository, instead of introducing new disconnected class

Reply:
Read about fake objects in more details [here](https://www.yegor256.com/2014/09/23/built-in-fake-objects.html)